simnet/alice/start.sh

```diff
@@ -1,5 +1,6 @@
-#!/bin/bash
+#!/bin/sh
 
 # Start alices node.
 # NOTE: (ccdle12) testing the autopilot feature.
-lnd --autopilot.active --autopilot.maxchannels=20 --autopilot.allocation=0.6 --rpclisten=127.0.0.1:10008 --rpclisten=0.0.0.0:10009 --datadir=./data --tlscertpath=./tls.cert --tlskeypath=./tls.key --listen=0.0.0.0:10011 --restlisten=localhost:8001 --debuglevel=debug --externalip=localhost:10011 --tlsextraip=0.0.0.0 --bitcoin.simnet --bitcoin.active --bitcoin.node=btcd --btcd.rpcuser=kek --btcd.rpcpass=kek
+# lnd --autopilot.active --autopilot.maxchannels=20 --autopilot.allocation=0.6 --rpclisten=127.0.0.1:10008 --rpclisten=0.0.0.0:10009 --datadir=./data --tlscertpath=./tls.cert --tlskeypath=./tls.key --listen=0.0.0.0:10011 --restlisten=localhost:8001 --debuglevel=debug --externalip=localhost:10011 --tlsextraip=0.0.0.0 --bitcoin.simnet --bitcoin.active --bitcoin.node=btcd --btcd.rpcuser=kek --btcd.rpcpass=kek
```

Comment:
We can probably remove this line for the time being

Reply:
oh yeah that's just a relic of the previous project, I'll remove it

Reply:
fixed!